pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/DSMConfig.java

```diff
@@ -23,6 +24,28 @@ public static String getSqlFromConfig(@NonNull String queryName) {
         return config.getString(queryName);
     }
 
+    public static Integer getIntFromConfig(@NonNull String name) {
```

Comment:
Take a look at ConfigManager.getInstance().  I think these methods could be replaced with calls to `ConfigManager.getInstance().getConfig().getInt()` and `getDouble()`.